GeoLlama/calc_by_slice.py

```diff
@@ -330,73 +321,23 @@ def evaluate_slice(slice_coords: list,
     cell_vecs = eigenvecs * rectangle_dims.reshape((2, 1))
 
     # Centralise lamella centroid to middle of slice along long axis
-    lamella_centre = centroid_s3
-
-    extrema = np.array(view.shape) - 1
-    ref_pt1 = lamella_centre + cell_vecs[1]
-    ref_pt2 = lamella_centre - cell_vecs[1]
+    lamella_to_slice_dist = 200 * max( np.abs(lamella_centre/np.array(view.shape) - 0.5) )
```

Comment:
where did 200 come from?

Reply:
I'm calculating the percentage of shift in terms of the _semi-major_ axis of the 2D image, not the full major axis, so the 200 actually comes from 100 / 0.5.